presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriter.java

```diff
@@ -180,6 +202,64 @@ public void commit()
                 throw new PrestoException(HIVE_WRITE_VALIDATION_FAILED, e);
             }
         }
+
+        List<ColumnStatistics> columnStatistics = new ArrayList<>();
+        for (ColumnStatistics columnStats : orcWriter.getFileStats()) {
+            if (columnStats.getMin() == null) {
+                continue;
+            }
+            columnStatistics.add(columnStats);
+        }
+
+        List<Type> types = new ArrayList<>(Arrays.asList(VARCHAR, VARCHAR, VARCHAR, BIGINT, BIGINT));
```

Comment:
We need to build the types based on the types of the table columns.

Reply:
These are for the `file_prefix`, `file_name`, `file_suffix`, `rows`, `bytes` of the manifest. These are fixed types irrespective of the table columns right ?